packages/nextjs/src/client/index.tsx

```diff
@@ -16,21 +16,33 @@ type NextClerkProviderProps = {
   Partial<PublishableKeyOrFrontendApi>;
 
 export function ClerkProvider({ children, ...rest }: NextClerkProviderProps): JSX.Element {
-  // @ts-expect-error
   // Allow for overrides without making the type public
-  const { authServerSideProps, frontendApi, publishableKey, proxyUrl, __clerk_ssr_state, clerkJSUrl, ...restProps } =
-    rest;
+  const {
+    // @ts-expect-error
+    authServerSideProps,
+    frontendApi,
+    publishableKey,
+    proxyUrl,
+    domain,
+    isSatellite,
+    // @ts-expect-error
+    __clerk_ssr_state,
+    clerkJSUrl,
+    ...restProps
+  } = rest;
   const { push } = useRouter();
 
   ReactClerkProvider.displayName = 'ReactClerkProvider';
 
   return (
+    // @ts-expect-error
```

Comment:
:upside_down_face: Better add this above each line that might raise the error? This way we'll know what needs to be changed in the future. Feel free to ignore.